build/test.mk

```diff
@@ -25,7 +25,7 @@ test: test-only
 test-only: test-unit test-integration
 
 test-unit:
-	@echo "=== $(PROJECT_NAME) === [ test-unit        ]: running unit tests..."
+	@echo "=== $(PROJECT_NAME) ${PROJECT_VER_TAGGED} === [ test-unit        ]: running unit tests..."
```

Comment:
Why add the version tag (and break the beautiful output format followed by the rest of the make system?

Reply:
I wasnt done. I should've opened a draft PR lol